src/main/java/com/meilisearch/sdk/Search.java

```diff
@@ -70,6 +70,12 @@ SearchResult search(String uid, String q) throws MeilisearchException {
      * @throws MeilisearchException Search Exception or Client Error
      */
     Searchable search(String uid, SearchRequest sr) throws MeilisearchException {
+        if (sr != null
+                && sr.rankingScoreThreshold != null
+                && (sr.rankingScoreThreshold > 1.0 || sr.rankingScoreThreshold < 0)) {
+            throw new MeilisearchException(
+                    "Ranking score threshold must be between 0 and 1.0 inclusive");
+        }
```

Comment:
No need to handle the error here, Meilisearch will return the appropriate error

Reply:
done